# this is a msg from the manager
bool flag
float64 obs_r
float64 infl_dist
float64 obs_x
float64 obs_y
float64 agent_x
float64 agent_y
